src/components/moneyTransaction/list.js

```diff
@@ -1,33 +1,40 @@
-import React from 'react'
+import React, { Suspense } from 'react'
+import { serverTimestamp } from 'firebase/firestore'
 import styles from '../../styles/moneyTransaction.module.css'
 import { updateTransaction } from '../../utils/transactions'
-import { ListItem } from './listItem'
 
-export const List = ({ transactions, setTransactions }) => {
+const LItem = React.lazy(() => import('./listItem'))
```

Comment:
I think I wouldn't create small code bundles like this. I'd rather add them at the route level (eg. SignIn/SignUp/MoneyTransactionList) get their own bundle

Reply:
@webpapaya Refactor 4.5/5
Da bin ich mir nicht 100% sicher. Ich glaub Suspense ladet die List Items dann schon "im nach hinein" erst und eben nicht sofort - soll bei grossen listen von Vorteil sein. (+optional/bessere Variante mit einem Intersection Observer)